_posts/blog/2018-10-17-ReviewGarcíaMárqu

```diff
@@ -0,0 +1,63 @@
+ 
+#Gabriel García Márquez. Harry Ransom Center. The University of Texas. Web.16 de Octubre 2018.https:[//hrc.contentdm.oclc.org/digital/collection/p15878coll51/]
```

Comment:
Please correct the formatting by adding spaces after the "3" and each period.

Reply:
Please follow markdown formatting for the url: put the entire URL in square brackets [ and add the url again in parentheses to create a link.